src/frontend/apps/e2e/__tests__/app-desk/teams-create.spec.ts

```diff
@@ -80,4 +80,16 @@ test.describe('Teams', () => {
     await panel.locator('li').getByText(teamName).click();
     await expect(elTeam).toBeVisible();
   });
+
+  test('checks alias teams url with homepage', async ({ page }) => {
+    const buttonCreateHomepage = page.getByRole('button', {
```

Comment:
nit:
```suggestion
    await expect(page).toHaveURL(/);
```
What do you think ?
It makes explicit the test assert the `/` and `/teams` share the same content